bespin_k8s_gcloud_cluster/tasks/get-service-account-credentials.yml

```diff
@@ -0,0 +1,29 @@
+- name: "Fetch details about bespin-deploy service account"
+  command: kubectl get serviceaccount bespin-deploy -o json
+  register: bespin_deploy_info
+
+- set_fact:
+    service_account_details: "{{ bespin_deploy_info['stdout'] }}"
+
+- name: Fetch details about bespin-deploy service account secret
+  command: >
+    kubectl get secret
+    "{{ service_account_details['secrets'][0]['name'] }}"
+    -o json
+  register: bespin_deploy_secret_info
+
+- set_fact:
+    service_account_secret_details: "{{ bespin_deploy_secret_info['stdout'] }}"
+
+- set_fact:
+    gcloud_k8s_token: "{{ service_account_secret_details['data']['token'] | b64decode }}"
+
+- set_fact:
+    gcloud_k8s_verify_ssl: no
```

Comment:
Can you elaborate why ssl verification must be disabled, somewhere in the code? We should have a good reason for turning it off - because of the obvious security implications, but also because that's requiring the development version of ansible. I also don't want this decision to be forgotten - if there comes a time when we don't need to disable ssl verification, I'd like it to be easy to tell and know we can turn it back on. It's also not clear how self-signed certs will impact it.

P.S. I notice that my `~/.kube/config` does _not_ set `insecure-skip-tls-verify` for any of my GKE clusters, so I'm curious if that's the same setting.